src/web3/EventPoller.ts

```diff
@@ -10,6 +10,14 @@ import { Client } from './Client';
 export const ON_MASSA_EVENT_DATA = 'ON_MASSA_EVENT';
 export const ON_MASSA_EVENT_ERROR = 'ON_MASSA_ERROR';
 
+/**
+ * Sorts slots by thread and period
```

Comment:
This function us not sorting but comparing slot. If you agree, can you create an issue to fix it's name ?

Reply:
issue created